schemas/meshTriangleCount.schema.json

```diff
@@ -7,113 +7,163 @@
   "properties": {
     "beard": {
       "type": "integer",
+      "minimum": 1,
       "maximum": 1000,
       "errorMessage": {
+        "minimum": "Too few triangles! This asset must have at least 1 triangle.",
         "maximum": "Mesh ${1/name} exceeds triangle count budget. Allowed: 1000. Found: ${0}."
       }
     },
     "body": {
       "type": "integer",
+      "minimum": 1,
       "maximum": 14000,
       "errorMessage": {
+        "minimum": "Too few triangles! This asset must have at least 1 triangle.",
         "maximum": "Mesh ${1/name} exceeds triangle count budget. Allowed: 14000. Found: ${0}."
       }
     },
```

Comment:
body subschema is not in use anymore, as it's not referenced by the outfit schema anymore, right? Should we remove it?